src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -179,12 +220,12 @@
             handleCancel() {
                 this.$emit("close")
             },
-            lastModified: function (date: string) {
-                return formatDateTime(date)
+            translate(text) {
+                return i18next.t(text, { lng: this.currentLanguage });
             },
             setDefaultCheckedItems: function () {
-                this.uploadFilesToAdr = outputFileTypes
-                    .filter(key => this.uploadFiles.hasOwnProperty(key))
+                this.uploadFilesToAdr = [...outputFileTypes, ...inputFileTypes]
+                    .filter(key => this.uploadableFiles.hasOwnProperty(key))
```

Comment:
I guess time will tell whether selecting all files by default rather than none) is the most helpful behaviour - I don't know whether users would generally want to upload a minority or the majority of the files.

Reply:
I guess if it becomes a hassle in future, we could add 'select all/deselect all buttons'.